test/accessibility.test.js

```diff
@@ -42,21 +36,10 @@ describe('accessibility', () => {
     test(path, async () => {
       const page = await browser.newPage();
       await page.goto(`http://localhost:${port}${path}`);
-      await page.addScriptTag({ path: require.resolve('html_codesniffer/build/HTMLCS.js') });
-      const messages = await page.evaluate(
-        () =>
-          new Promise((resolve) => {
-            window.HTMLCS.process('WCAG2AA', window.document.body, () => {
-              resolve(window.HTMLCS.getMessages());
-            });
-          }),
-      );
-      for (const message of messages) {
-        if (message.type === MESSAGE_TYPE_ERROR) {
-          throw message;
-        }
-      }
+      const results = await new AxePuppeteer(page).withTags(['wcag2a', 'wcag2aa']).analyze();
```

Comment:
This is "bare minimum". We could / should choose to omit `.withTags(['wcag2a', 'wcag2aa'])` which identifies several more "best practice" failures, but we can improve that iteratively.

```suggestion
      const results = await new AxePuppeteer(page).analyze();
```